standard/info/wikis/fighters/info.lua

```diff
@@ -648,6 +648,19 @@ return {
 				lightMode = 'KILL la KILL - IF default allmode.png',
 			},
 		},
+		koihime = {
+			abbreviation = 'Koihime Enbu',
+			name = 'Koihime Enbu',
+			link = 'Koihime Enbu',
+			logo = {
+				darkMode = 'Koihime Enbu logo.png',
```

Comment:
could we make this follow file standards naming?
(if it fulfills file standards in general)

Reply:
I fixed this on behalf of Ade. Resolving comment